config/locales/en.yml

```diff
@@ -1814,7 +1814,17 @@ en:
   mail_body_account_information: "Your account information"
   mail_body_account_information_external: "You can use your %{value} account to log in."
   mail_body_lost_password: "To change your password, click on the following link:"
-  mail_body_register: "To activate your account, click on the following link:"
+  mail_body_register: "Welcome to OpenProject. Please activate your account by clicking on this link:"
+  mail_body_register_header_title: "Project member invitation email"
+  mail_body_register_user: "Dear %{userName}, "
+  mail_body_register_youtube_channel: "Please feel free to browse our youtube channel ( "
```

Comment:
Breaking up this sentence works fine in english but it might not work the same way in other languages.
You could probably just add a single multiline string like so:

```
mail_body_register_links_html: |
  Please feel free to browser our YouTube channel (%{youtube_link}) where we provide a webinar (%{webinar_link}) ... 
  This text can be writte in multiple lines as well.
  </br>
  You can add newlines to it
  This will all end up as one single string
```

The `_html` is important so that if you add links as variables, they are being picked up

In the html template, you would then use it like this:

```
<%= t(:mail_body_register_links_html,
            webinar_link: link_to("Webinar Link", OpenProject::Static::Links....),
            youtube_link: link_to('Youtube Link', OpenProject::Static::Links....) ....
%>
```

And in the text email just use the links itself
```
<%= t(:mail_body_register_links_html,
            webinar_link: OpenProject::Static::Links....),
            youtube_link: OpenProject::Static::Links....) ....
%>
```